subnets.tf

```diff
@@ -2,36 +2,36 @@ resource "aws_db_subnet_group" "rds_subnet_group" {
   name = "${var.db_name}-rds-subnet-group"
 
   subnet_ids = [
-    "${aws_subnet.db_subnets.*.id}",
-    "${var.primary_db_subnets}",
+    aws_subnet.db_subnets.*.id,
+    var.primary_db_subnets,
   ]
```

Comment:
Did this pass `terraform validate`? I think `flatten()` is need here to make the terraform parser happy.

Reply:
you are correct, I've included flatten now